src/openforms/prefill/contrib/haalcentraal/plugin.py

```diff
@@ -72,9 +72,14 @@ def _get_values_for_bsn(cls, bsn: str, attributes: Iterable[str]) -> Dict[str, A
     def get_prefill_values(
         cls, submission: Submission, attributes: List[str]
     ) -> Dict[str, Any]:
-        if not submission.bsn:
+        if (
+            not hasattr(submission, "auth_info")
+            or submission.auth_info.attribute != AuthAttribute.bsn
+        ):
+            #  If there is no bsn we can't prefill any values so just return
+            logger.warning("No BSN associated with submission, cannot prefill.")
```

Comment:
```suggestion
            logger.info("No BSN associated with submission, cannot prefill.")
```
This situation probably happens on test environments, I think we can use a lower logging level.